crates/store/re_video/src/decode/ffmpeg_h264/ffmpeg.rs

```diff
@@ -93,6 +93,11 @@ impl From<Error> for crate::decode::Error {
 /// ffmpeg does not tell us the timestamp/duration of a given frame, so we need to remember it.
 #[derive(Clone)]
 struct FfmpegFrameInfo {
+    /// The start of a new [`crate::demux::GroupOfPictures`]?
+    ///
+    /// If true, an entire frame can be decoded from this one sample,
```

Comment:
I thought for very long that this was true. But now I'm not so sure anymore that we are guaranteed to get a frame back from a decoder when we enqueue such a frame.
I would have expected PTS==DTS on those, but that part I know for sure now is not the case.

One subtle problem with this statement that I'm a bit more sure about is that while we have a `sample -> frame`, it may still be that a frame needs meta information from other samples even though this is supposed to be an IDR frame.
For instance in AV1, the information we append right now in our annex_b stream in the ffmpeg decoder is dispersed over the samples of the video and I have no ideas if there's any guarantees at all.


With that in mind I'd be in favor for leaving this out of the doc comment & tooltip (affects a bunch of places where this is copy pasted) until we understand better what we're dealing with.

Reply:
FWIW, this is what Claude 3.5 believes:

![image](https://github.com/user-attachments/assets/f20389ed-0bae-41ac-b534-79a049ffb0cc)

…but I'm sure it's mostly trained on common misconceptions

Reply:
not a strong conviction here as described.
What bugs me about all this is that the `is_sync` is derived in quite strange way in the mp4 crate which I don't follow and I'd rather have per codec defined things that are easier to look up :(

Reply:
we obviously depend on it meaning _something_ though as it governs our enqueuing strategy

Reply:
maybe just slap a `We generally assume` in front of it 🤷